README.md

```diff
@@ -2,6 +2,7 @@
 
 [![CircleCI](https://circleci.com/gh/Seneca-CDOT/telescope.svg?style=svg)](https://circleci.com/gh/Seneca-CDOT/telescope)
 [![js-airbnb/prettier-style](https://img.shields.io/badge/code%20style-airbnb%2Fprettier-blue)](https://github.com/airbnb/javascript)
+![Node.js CI](https://github.com/Seneca-CDOT/telescope/workflows/node-js-ci/badge.svg)
```

Comment:
I think you're using the `workflow name`, method, right?
According to `github`'s docs, I think you're missing the word `action` in your URL.
Their example:
> github.com/**actions**/hello-world/workflows/Greet%20Everyone/badge.svg

Reply:
In this example of theirs, I believe `actions` refers to their repository owner, followed by `hello-world`, which is the repository name.
Therefore, I replaced `actions` with `Seneca-CDOT` and `hello-world` with `telescope`.

Reply:
My bad, I just realized about the word `action` being the organization.
I tested it with a different repo and it seems to work (a repo inside of `Seneca-CDOT` that uses actions to check for linting errors). I wonder if it's because of something we're not doing with Actions
![image](https://user-images.githubusercontent.com/23108901/98879486-aab7f480-2453-11eb-8311-7b24c7a0e88f.png)

Reply:
No need to apologize! I tried putting action in there myself before going back and reading the documentation. It seemed like it would fit.

I attempted to test the badge on my own branch, and I was able to get a `passing` status through [shield.io](https://shields.io/category/build):

![image](https://user-images.githubusercontent.com/37879416/98879861-7f81d500-2454-11eb-805a-963f1ba9ccac.png)

Reply:
I did a bit more digging:
Testing my repo I tried the `github`'s method and the post's method, like this:
[![YourActionName Actions Status](https://github.com/manekenpix/telescope/workflows/node-js-ci/badge.svg)](https://github.com/manekenpix/telescope/actions)
https://github.com/manekenpix/telescope/workflows/node-js-ci/badge.svg https://github.com/manekenpix/telescope/actions

![Node.js CI](https://github.com/manekenpix/telescope/workflows/node-js-ci/badge.svg?branch=test-ci-badge)
https://github.com/manekenpix/telescope/workflows/node-js-ci/badge.svg?branch=test-ci-badge

I created a PR and I merged it in my own repo, and it seemed to work (you can see they're both green). I also changed `node-js-ci.yml` and I added this:
```yaml
push:
    branches:
      - master
```
in the `on` section to trigger `Actions` on `master`. This could be one of the reasons why we don't get the expected result. _Technically_, we've never run `Actions` on master, so the badge has nothing to show.

Reply:
We should get it to run on merges to `master` too, nice find.

Reply:
From what I've read, there isn't a way to specify merges, and that they just get lumped in with `on: push:`
https://github.community/t/trigger-workflow-only-on-pull-request-merge/17359/3

Testing the badge on my own repo now, hopefully I never see this red fail again haha

Reply:
Still trying to work out how I'm stuck in no status...
This is the branch I'm working through right now: https://github.com/matt-ross16/telescope/tree/workflowbadge-test

I made the adjustments to `node-js-ci.yml` to add the `push` action, but callint the badge still leaves me with `No status`

[![node-js-ci](https://github.com/matt-ross16/telescope/workflows/node-js-ci/badge.svg?branch=workflowbadge-test)](https://github.com/matt-ross/telescope/actions)
https://github.com/matt-ross16/telescope/workflows/node-js-ci/badge.svg?branch=test-ci-badge

Reply:
@manekenpix I tried recreating the Workflow file to get the badge showing the proper status, but on my branch (above) I have had no luck... any thoughts?

Reply:
@matt-ross16 your `node-js-ci.yml` has another branch in `pull_request` and `push` called `workflow-test` but the branch you're testing is called `workflowbadge-test`. Could that be the reason?
Also, your URL shows this:
>![image](https://user-images.githubusercontent.com/23108901/99156769-a842de00-2691-11eb-9fc1-4573b9013c74.png)

What's `test-ci-badge`? shouldn't it be `workflowbadge-test`?

Reply:
Oops my bad on the URL... I forgot to change the branch name from your previous one.
Thanks for catching the `.yml` discrepancy as well! Can't believe that I forgot to include `badge`

Testing it out again real quick and looking over all the links and files to make sure there aren't more mistakes, but so far no change

Reply:
@matt-ross16 the status of the badge that you posted here in the comments just changed, I think you fixed it :)

>![image](https://user-images.githubusercontent.com/23108901/99158796-d4b42580-26a4-11eb-952c-698c4cd95739.png)

Reply:
Thanks for pointing this out! It must have taken a bit of time for it to actually change statuses! Getting the PR prepped with the proper settings now